js/pages/myToDoList.js

```diff
@@ -650,6 +649,12 @@ const setModuleAttributes = (data, modules, collections) => {
     modules['Enter SSN'].hasIcon = false;
     modules['Enter SSN'].noButton = false;
     modules['Enter SSN'].estimatedTime = 'Less than 5 minutes'
+    
+    modules['Covid-19'].header = 'COVID-19'
+    modules['Covid-19'].description = 'This survey asks questions about your history of COVID-19, including any vaccinations you may have received and details about times you may have gotten sick with COVID-19.';
+    modules['Covid-19'].hasIcon = true;
```

Comment:
should this be true? I'm not sure how the `hasIcon` property is used to drive logic, but we aren't setting this true anywhere else